### ISSUE TITLE
LG G4 support

### ISSUE THREAD

**Issue by codeworkx** [almalence/OpenCamera#33]
The G4 supports the new API but LG itself is using the old API on their own Camera App.

Instead of Camera.open() use Camera.openLegacy(camId, android.hardware.Camera.CAMERA_HAL_API_VERSION_1_0)

Then set parameter "lge-camera" to "1".

After that you can control the lg iso modes with parameter "lg-iso". This parameter has much more values than the default "iso" parameter.

And to get the laser focus working you should set "focus-mode" to "continuous-video".

In short:
Camera.open() -> Camera2 -> QCom Hardware HAL3 (untouched by LG, very basic)
Camera.openLegacy() -> Legacy Camera -> QCom Hardware HAL2 (extended by LG, a lot of stuff to play with)
